lib/models/liturgical-day-def.ts

```diff
@@ -293,7 +293,7 @@ export class LiturgicalDayDef implements BaseLiturgicalDayDef {
         if (pointer.titles) {
           martyrologyItem.titles = this.#combineTitles(pointer.titles, pointer.id, previousDef);
         }
-      }
+      } // eslint-disable-line @typescript-eslint/brace-style
       // If the Martyrology item is not found, it means this item is badly referenced in the date definition.
       // In this situation, romcal must report en error.
       // Note: romcal do not report an error when the liturgical day ID is used to find a martyrology item,
```

Comment:
Don’t apply this suggestion via web UI, as I could not select all L300-301 which should be replaced by this suggestion.

```suggestion

        // If the Martyrology item is not found, it means this item is badly referenced in the date definition.
        // In this situation, romcal must report en error.
        // Note: romcal do not report an error when the liturgical day ID is used to find a martyrology item,
        // because this liturgical day definition may not be related to a martyrology item.
      } else if (input.martyrology) {
```